django_stomp/execution.py

```diff
@@ -149,3 +150,15 @@ def _create_queue(queue_name: str, durable_topic_subscription: bool = False, rou
 def _create_dlq_queue(destination_name: str):
     dlq_destination_name = create_dlq_destination_from_another_destination(destination_name)
     _create_queue(dlq_destination_name)
+
+
+def _get_or_create_correlation_id(payload: Payload) -> str:
+    if "correlation-id" in payload.headers:
+        return payload.headers["correlation-id"]
+
+    if not is_correlation_id_required:
+        correlation_id = uuid.uuid4()
+        logger.info(f"STOMP_CORRELATION_ID_REQUIRED was set to false. New correlation-id was generated {correlation_id}")
```

Comment:
Is `warning` the case to be used here?

Reply:
`warning` can be used in alerts, as it is the most `info` ... no?

Reply:
That is the idea. In context of `django-stomp` suggestions, using this key is not the standard and is discouraged. I even think whether this approach is good, because every single message will have a log printed. Evolving more this discussion, I believe it can be warned only once before `callback` processing.

Reply:
Thinking better about it, I agree with the suggestion to change to warning. The flag will be turned off rationally, for this reason we don't need to tell it to him all the time.

Reply:
If the problem is that it'll print a message ever time that `callback` is called (ever time a new message frame is received by the consumer), then setting it to `warning` will not resolve. Unless the application log level is set to `ERROR` or `CRITICAL`.

But, as I can see, the real purpose of this log message is to print the generated `correlation_id`, couldn't this be a `debug`? And for the message that is alerting the feature disable, as @willianantunes suggested, to warn it only once, it could be placed right before the `_create_dlq_queue` or after `logger.info("Starting listener...")`

Reply:
I think @fabiohk has reason, the message is a `debug`, as the `correlation-id` is information to be used in tshoot.

About message `STOMP_CORRELATION_ID_REQUIRED was set to false.`, I think it should be recorded only once, as the parameter will be intentionally changed.